### PR TITLE
Spruce up the README

### PR DESCRIPTION
**Pull request by mbestavros**
Here is a description.

### CODE REVIEW

**Comment by npmccallum** [comment]
This is a comment to see how events work. Note that this PR is not mergeable.

**Review by npmccallum — CHANGES_REQUESTED** [review]
This needs changes.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # bot-test
-Tests for the bot
+A test repository for the Enarxbot.
```
Comment (mbestavros) [inline README.md]: This is a code quote.

**Review by npmccallum — CHANGES_REQUESTED** [review]
Reject a second time.

**Review by npmccallum — CHANGES_REQUESTED** [review]
A bucket full of denials!

**Review by npmccallum — CHANGES_REQUESTED** [review]
Run actions again.

**Comment by mbestavros** [comment]
It may be helpful to leave this open for GraphQL testing. I'm not closing this for the moment.

**Pull request closed without merging**